gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-details/api-tab-details.component.html

```diff
@@ -34,19 +34,16 @@
 
   <div>
     <mat-card appearance="raised" class="api-details__tab-details-info__card">
-      @if (api.version) {
-        <div>
-          <div class="m3-body-large" i18n="@@apiDetailsVersion">Version</div>
-          <div class="m3-body-medium">{{ api.version }}</div>
-        </div>
-      }
-
-      @if (api.updated_at) {
-        <div>
-          <div i18n="@@apiDetailsLastUpdated" class="m3-body-large">API Last Updated</div>
-          <div class="m3-body-medium">{{ api.updated_at | date }}</div>
-        </div>
-      }
```

Comment:
Shouldn't we always be displaying these?

Reply:
But these values are hardcoded.
I thought the whole point of this story is to make them dynamic.
For example: what if user removed api.version, or made it last, not first.
That's why we don't need these hardcoded values

Reply:
For the older portal, we will always show certain values in the API Details sidebar. Which made me wonder if we want to still show those basic fields in any case. Did you get a chance to speak with Anthony or Audrey?

Reply:
Resolved